source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md

```diff
@@ -0,0 +1,104 @@
+---
+favorite: true
+id: 42
+title: Start and Stop Your Work Day With a List
+---
+
+I've developed a habit that's at the heart of my personal operating system: I
+start and stop my work day with a list. Let's talk about what this means and how
+it's improved my work life.
+
+## What Does This Even Mean?
+
+It means that when I sit down at my desk to start working the first thing I do
+is open [Todoist][] and find a recurring item called "run start list". When I
+find it, I click on it and see a bunch of subtasks:
+
+* add times to [Forty][]
+* drill master passwords
+* prepare stop list
+* start standup notes
+* open dashboards
+* add meetings here
+* say hello in Slack
+* read through Slack
+* process email
+* review GH notifications
+* review pulls
+* review Jira
+
+This list is what I follow to begin my day and it goes from least distracting to
```

Comment:
totally non-blocking - distracting has a pretty negative connotation - do you mean "shallowest to deepest" or maybe something else?

Reply:
Yep, this wasn't quite what I meant - I have a rewording that I'm finishing up. 👍